product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx

```diff
@@ -341,30 +342,31 @@ __OUTPUT__
 Pause Cluster operation succeeded, "p-94pjd2w0ty"
 ```
 
-### Resuming a cluster
+### Resume a cluster
 
 To resume a cluster, use the `cluster resume` command. The `cluster resume` command supports `flag` and `interactive` mode. The syntax for the command is:
 
 ```
 biganimal cluster resume {--id | --provider --region --name}
 ```
 
-Where `id` is a valid cluster ID. The `id` is mandatory.
-      `provider` is a cloud provider of the cluster.
-      `region` is the region of the cluster.
-      `name` is the name of the cluster.
+Where:
+- `id` is a valid cluster ID. The `id` is mandatory.
```

Comment:
@nidhibhammar again, this doesn't seem to be mandatory if you can instead use provider/region/name.